src/main/java/gov/cabinetoffice/gap/adminbackend/repositories/GrantExportRepository.java

```diff
@@ -38,4 +38,9 @@ void updateExportRecordLocation(@Param("submissionId") UUID submissionId,
 
     Long countByIdExportBatchIdAndStatusNot(UUID exportGrantId, GrantExportStatus status);
 
+    @Query("select g from GrantExportEntity g where g.id.exportBatchId = ?1 and g.status = ?2")
```

Comment:
nit: don't really need a query here, as the name of the method reflect what jpa would do